scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -322,7 +280,7 @@ function verify_subm_routeagent_pod() {
   for subm_routeagent_pod_name in "${subm_routeagent_pod_names_array[@]}"; do
     echo "Testing Submariner routeagent pod $subm_routeagent_pod_name"
     kubectl get pod $subm_routeagent_pod_name --namespace=$subm_ns -o json
-    kubectl get pod $subm_routeagent_pod_name --namespace=$subm_ns -o jsonpath='{.spec.containers..image}' | grep $subm_routeagent_image_repo:$subm_engine_image_tag
```

Comment:
for the engine we also do submariner:$subm_engine_image_tag so I removed the variable and set it here too..